backend/loernwerkUtilities.ts

```diff
@@ -69,30 +112,11 @@ declare module 'express-session' {
     }
 }
 
-/**
- * Custom error class allowing to specify the error code in the constructor.
- */
-export class LoernwerkError extends Error {
-    code: LoernwerkErrorCodes;
-
-    /**
-     * Constructs a new error with the given message and error code.
-     * @param message Message of the error
-     * @param code Error code
-     */
-    constructor(message: string, code: LoernwerkErrorCodes) {
-        super(message);
-        this.code = code;
+// Declaring additional request properties for H5P
+declare module 'express-serve-static-core' {
+    interface Request {
+        user: IUser;
+        t: (errorId: string, replacements: { [key: string]: string }) => string;
```

Comment:
t is a really bad name

Reply:
Sadly, forced by H5P library

Reply:
then the comment below applies even more, documentation needed. maybe even add this comment to the doc